Adding links to chainspec file and gas limit details (#673)
casper-network/docs: source/docs/casper/glossary/C.md

## Chainspec {#chainspec}

> A collection of configuration settings describing the state of the system at genesis and upgrades to basic system functionality (including system contracts and gas costs) occurring after [genesis](G.md#genesis). Changes to the [chainspec](https://github.com/casper-network/casper-node/blob/release-1.4.8/resources/production/chainspec.toml) occurs depending on the system updates and releases.

Reviewer: ```suggestion
A collection of configuration settings describing the state of the system at genesis and upgrades to basic system functionality (including system contracts and gas costs) occurring after [genesis](G.md#genesis). Here is an example [chainspec](https://github.com/casper-network/casper-node/blob/release-1.4.8/resources/production/chainspec.toml), which will change with newer releases.
```
